src/components/ViewRequest/Actions/SendRequest.tsx

```diff
@@ -28,6 +30,25 @@ const SendRequest = () => {
     draftErrors = validateRequest(request.data);
   }
 
+  const readyForNextStage = currentStage?.readyForNext(request?.data) ?? false;
+
+  let disableSend = true;
+  if (currentStage?.next && readyForNextStage) {
```

Comment:
The "readyForNextStage" check here, prevents your nice dialog message from ever showing, since the button becomes disabled, instead of letting them click it, then getting the dialog message.